system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -196,7 +196,7 @@ impl pallet_timestamp::Config for Runtime {
 	/// A timestamp: milliseconds since the unix epoch.
 	type Moment = u64;
 	type OnTimestampSet = Aura;
-	type MinimumPeriod = ConstU64<{ SLOT_DURATION / 2 }>;
+	type MinimumPeriod = ConstU64<0>;
```

Comment:
is this a tmp value or is it going to be zero?

Reply:
IIUC, we don't need to set the minimum period anymore. With async backing at higher velocities, we may produce more blocks per slot, and nothing stops us from producing more than two blocks per slot, so this limit doesn't make sense, and I set it to zero here. But maybe some of the senior fellows will correct me.

Reply:
If that's the case then I need to figure out how to deal with it on chopsticks